backendImpl/src/test/java/com/virtuslab/gitmachete/backend/integration/StatusAndDiscoverIntegrationTestSuite.java

```diff
@@ -58,7 +54,7 @@ public static void doEnsureExpectedCliVersion() {
         SETUP_FOR_NO_REMOTES,
         SETUP_WITH_SINGLE_REMOTE,
         SETUP_WITH_MULTIPLE_REMOTES,
-        SETUP_FOR_DIVERGED_AND_OLDER_THAN,
+        //SETUP_FOR_DIVERGED_AND_OLDER_THAN,
```

Comment:
Pls open an issue on diagnosing & reverting `SETUP_FOR_DIVERGED_AND_OLDER_THAN` on Windows (currently the script is stalling) :/